src/zm_monitor.cpp

```diff
@@ -394,11 +396,11 @@ Monitor::Monitor(
              + 64; /* Padding used to permit aligning the images buffer to 16 byte boundary */
 
     Debug( 1, "mem.size=%d", mem_size );
-	mem_ptr = NULL;
```

Comment:
In the interest of keeping this huge PR as small and precise as possible, would you mind reverting the tab / spacing fixes?  Strictly, they aren't related to multi-server.